### ISSUE TITLE
Error: creating the captive portal failed.  Caused by "ipv4.addresses: this property cannot be empty for \'method=manual\'

### ISSUE THREAD

**Issue by tdoneal** [resin-io/resin-wifi-connect#222]
Hi, I'm trying to get started by testing resin-wifi-connect on my local rpi 3.  I'm not running from docker (yet), but I'm trying to just manually recreate the start.sh functionality.

I disabled dhcpd, installed and enabled NetworkManager, downloaded the wifi-connect executable (v4.1.2) and ran:

./wifi-connect

And got
```
WiFi device: wlan0
[wifi_connect::network:WARN] No access points found - giving up...
Starting access point...
[network_manager::dbus_api:ERROR] org.freedesktop.NetworkManager::AddAndActivateConnection method call failed on /org/freedesktop/NetworkManager
Error: Creating the captive portal failed
  caused by: D-Bus failure: org.freedesktop.NetworkManager::AddAndActivateConnection method call failed on /org/freedesktop/NetworkManager
  caused by: "ipv4.addresses: this property cannot be empty for \'method=manual\'"
```

Any idea what could be going on here?  It seems like the API call to NetworkManager is failing - which is bizarre.  Unless I'm somehow running the wrong version of NetworkManager?  My network manager version installed is 0.9.10.0-7

**Comment by majorz** [comment]
Yes, this is exactly so. To run WiFi Connect you will need a more recent version of NetworkManager. That is to say you need to use either Raspbian Stretch or install a more recent NetworkManager version. Please let me know if you have any further questions.

**Comment by tdoneal** [comment]
Great!  Yeah I tried it using an updated Raspbian (stretch) and it worked perfectly.  Leaving here in case someone else has same problem.  You can close this.

**Comment by jbrepogmailcom** [comment]
Hello. Tried on Raspberry 4 with Raspbian Buster and got nearly same error:

sudo ./wifi-connect -i wlan0
Targeted WiFi device: wlan0
[wifi_connect::network:WARN] No access points found - giving up...
Starting access point...
[network_manager::dbus_api:ERROR] org.freedesktop.NetworkManager::AddAndActivateConnection method call failed on /org/freedesktop/NetworkManager
Error: Creating the captive portal failed
  caused by: D-Bus failure: org.freedesktop.NetworkManager::AddAndActivateConnection method call failed on /org/freedesktop/NetworkManager
  caused by: D-Bus failure: Method call failed after 10 retries

**Comment by romulomanzano** [comment]
Hi - getting the exact same error, netrowk manager version installed is 1.14
